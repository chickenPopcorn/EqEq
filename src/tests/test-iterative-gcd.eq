Gcd = {

  gcd = {
    a= 10;
    b=20;
    while (a != b) {
      if (a > b) {a = a - b;}
      else {b = b - a;}
    }
    a;
   }
}

Gcd:find gcd {
  print("gcd between 10 and 20 is %0.f\n", gcd);
}
