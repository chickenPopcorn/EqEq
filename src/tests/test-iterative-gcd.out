gcd between 10 and 20 is 10
